cocli/data/corim/templates/corim-cca-realm.json

```diff
@@ -0,0 +1,25 @@
+{
+    "corim-id": "5c57e8f4-46cd-421b-91c9-08cf93e13cfc",
+    "dependent-rims": [
+        {
+            "href": "https://parent.example/rims/ccb3aa85-61b4-40f1-848e-02ad6e8a254b",
+            "thumbprint": "sha-256:5Fty9cDAtXLbTY06t+l/No/3TmI0eoJN7LZ6hOUiTXU="
+        }
+    ],
+    "profiles": [
+        "http://arm.com/cca/realm/1"
```

Comment:
FYI.  There was a discussion in the PSA Token draft regarding URIs used for naming profiles which deprecated the `http` scheme in favour of the `tag` scheme, to avoid accidental dereferencing.  I think the same logics would apply here.

Reply:
Thanks,  I follow that:
However, there should be some degree of consistency in our approach.
Otherwise the profile definition should look very much fragmented.

Either we change the same for CCA Token Profiles and CCA Platform Endorsement Profile at the same time 
and do it in one batch otherwise it is super confusing to everyone!

Reply:
Created Issue #110 to address this in future changes